distribution/src/conf/internal-apis.xml

```diff
@@ -33,6 +33,26 @@
                             </user>
                         </users>
                     </UserStore>
+                </handler-->
+                <handler class="org.wso2.micro.integrator.management.apis.security.handler.JWTTokenSecurityHandler"
```

Comment:
Shall we change the order of handlers since the default is going to be the JWT handler?

Reply:
Thanks changed the order